### PR TITLE
More backlog fixes

### PR DESCRIPTION
**Pull request by emoryjgrubbs**
i didn't change the styling of the scrollbars, but i think it's a lot better just without the horizontal scroll
no review modal yet, but i'm making a new issue for it

issue list (let me know if anything is unclear, i initially just listed these out as a todo list for myself)
------------
- scrollbars
- look at deals indicator
- inventory item cards are not aligned anymore, i think because the picture got larger
- change increment amount input to match search (either by removing select border or changing it and the lack of one on search)
- item action menu
- nav showing links it shouldn't when refreshing cookie
- dropdown transitions
- change submit to review changes + add change of color and pointer indication of "no source
- make the review button stick to bottom of page content, prevent it from going into footer
- align dropdown text left/add chevron to indicate status or that it is dropdown
- review button covering filter options on mobile

### CODE REVIEW

**Review by CrazEpic — APPROVED** [review]
good, few suggestions and concerns, see the comments

File: pages/inventory-management.vue [hunk 1]
```diff
@@ -23,6 +22,10 @@ div
         p( v-if="addItemsOpen" ).text-center.pt-20 Add Items
         CategoryItemsGrid.my-4(v-for="category in Object.keys(filteredCategoryItems)" :headingName="category")
             ItemCard(v-for="item in filteredCategoryItems[category]" @changeAmountUpdate="updateItemChangeAmount" :name="item.name" :imgURL="item.imgURL" :itemID="item.itemID" :currentCount="item.quantity" :changeCount="(item.itemID in inventoryCountChanges) ? inventoryCountChanges[item.itemID] : 0" :itemDeal="((item.Deal) ? {actualCount: item.Deal.actualCount, adjustedCount: item.Deal.adjustedCount}: {})" typeOfCard="INVENTORY")
+        // submit button
+        div.sticky.bottom-8.z-20.flex.justify-end
+            button( v-if="source === ''" ).button.bg-red-negative.text-white.w-full.sm_w-60.drop-shadow-standard.cursor-not-allowed No Source
```
[the remark this answers is not in the document]
Comment (emoryjgrubbs) [reply pages/inventory-management.vue:27]: i'll make sure it fits, but changing it to no source selected should make it more clear

File: components/ItemCard.vue [hunk 2]
```diff
@@ -1,52 +1,72 @@
...
-      button.bg-utd-green.text-white.shadow-md.rounded-full.p-2.w-10.h-10.flex.items-center.justify-center(@click="$emit('changeAmountUpdate', itemID, changeInput)")
-        span.text-xl.font-bold +
-
-    div(v-if="typeOfCard === 'SHOPPING'").mt-4
-      button.bg-utd-green.text-white.py-1.rounded-full.w-full.font-semibold(@click="addToCart") Add to Cart
-
-    div.mt-3.text-center
-      p.text-xl {{ name }}
+    div.w-72.relative
+        <!--Deal-->
```
[the remark this answers is not in the document]
Comment (emoryjgrubbs) [reply components/ItemCard.vue:3]: for the comments?

File: components/ItemCard.vue [hunk 3]
```diff
@@ -1,52 +1,72 @@
...
+                    enter-from-class="transform scale-95 opacity-0"
+                    enter-to-class="transform scale-100 opacity-100"
+                    enter-active-class="transition duration-100 ease-out pointer-events-none"
+                    leave-from-class="transform scale-100 opacity-100"
+                    leave-to-class="transform scale-95 opacity-0"
+                    leave-active-class="transition duration-75 ease-in pointer-events-none"
+                )
+                    MenuItems.absolute.top-16.right-2.bg-white.bg-opacity-90.drop-shadow-standard.rounded-md.py-2.px-6
+                        MenuItem
+                            div.flex.justify-center.items-center.px-1.py-1.cursor-pointer.hover(@click="editItem").border-b.border-gray-400.text-lg.font-normal Edit
```
[the remark this answers is not in the document]
Comment (emoryjgrubbs) [reply components/ItemCard.vue:29]: i don't think those do anything, i'll go in and clean them up/make sure there aren't any other parenthesis in weird places

**Pull request merged**